### PR TITLE
Issue #9 Create water

### LINKED ISSUE #9: Create Water
A user wants to add a new Water for Analysis.

Acceptance  Criteria
Implement the design for add Water UI
Implement the logic to add a new Water
Write the necessary unit test

Implementation Note
Use a standard design for the UI

### PR DESCRIPTION
**Pull request by denkyipatrick**

### CODE REVIEW

**Review by EugeneAgyemang — APPROVED** [review]
I finished Reviewing this PR and everything works fine.
Just a validation to avoid numbers and special characters to be added

File: FPIS/Views/CreateWaterForm.cs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        public void ValidateWaterName()
+        {
+            string waterName = materialTextBoxWaterName.Text.Trim();
+
+            if (waterName.Length == 0)
+            {
+                this._isDataValid = false;
+                materialLabelWaterNameError.Text = "Water name is required!";
+            }
+        }
```
Comment (EugeneAgyemang) [inline FPIS/Views/CreateWaterForm.cs:52]: Kindly add a validation to avoid numbers and special characters

**Review by EugeneAgyemang — CHANGES_REQUESTED** [review]

File: FPIS/Views/CreateWaterForm.cs (hunk 1 above)
Comment (denkyipatrick) [reply FPIS/Views/CreateWaterForm.cs:52]: Validation is added

**Review by EugeneAgyemang — APPROVED** [review]

**Review by EugeneAgyemang — APPROVED** [review]
Finished Reviewing and everything works fine

**Pull request merged**